rollup.config.js

```diff
@@ -15,7 +16,54 @@ export function simpleReplace(replacements) {
   };
 }
 
+function inlineHtml(htmlOutPath) {
```

Comment:
can you add comments here, or below where it's used, on what this is for?

Reply:
Done.